legend.go

```diff
@@ -83,12 +83,12 @@ func makeLegend() (Legend, error) {
 // draw draws the legend to the given draw.Canvas.
 func (l *Legend) draw(c draw.Canvas) {
 	iconx := c.Min.X
-	textx := iconx + l.ThumbnailWidth + l.TextStyle.Width(" ")
-	xalign := 0.0
+	sty := l.TextStyle
+	textx := iconx + l.ThumbnailWidth + sty.Width(" ")
 	if !l.Left {
 		iconx = c.Max.X - l.ThumbnailWidth
 		textx = iconx - l.TextStyle.Width(" ")
-		xalign = -1
+		sty.XAlign -= 1.
```

Comment:
Why did this change from `= -` to `-=`?

Also, why the stray dot in `1.`? I'd prefer `1.0` to the stay dot, but I'm perfectly happy with just plane old `1`.